### DISCUSSION TITLE
Wrong rotation

### DISCUSSION THREAD

**Discussion by blissi** [jomjol/AI-on-the-edge-device#3377] [General] [answered]
Hi,
my ESP32 doesn't recognize the counter values anymore.
I had the 16.0.0-RC1 release (where it once worked, but then not anymore) and now I updated to 16.0.0-RC4 and it still doesn't work.
The problem is that the image is not rotated properly:
![image](https://github.com/user-attachments/assets/e859f6dd-bb45-40b1-8f60-79550682a600)

I have -90° set up and in my Digit ROI configuration the image is displayed properly:
![image](https://github.com/user-attachments/assets/eb42575d-75c1-481b-9400-e6883dbd1fb9)

Do you have an idea what's wrong here?
Is this a known problem?

**Comment by agusserraalh** [comment, marked as answer]
What happens is that in the alignment process it does not find the reference images, so it does not frame it correctly. Look for unique figures in the image and with very good contrast. I would improve the lighting.

**Comment by blissi** [comment]
Thank you, that helped :-)
